packages/trader/src/Modules/Account/Sections/Security/ApiToken/clipboard.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover, Icon } from '@deriv/components';
+import { copyToClipboard } from '_common/utility';
+import { localize } from '@deriv/translations';
+
+const Clipboard = ({ token }) => {
```

Comment:
I think there's a Clipboard component that's used in the MT5 module. Can we reuse that?

Reply:
lets put that into component?, or is it possible to import from mt5 module?

Reply:
Better avoid importing from other modules. makes it harder in the future to refactor things. Can we move it to components and then reuse?